README.md

```diff
@@ -29,11 +37,9 @@ Import-Module -Name '.\Carbon.Management'
 
 # The Carbon Family
 
-* `Carbon`: the original. This is where all the other Carbon modules started from. It is great and one of the most
+* [Carbon](http://get-carbon.org): the original. This is where all the other Carbon modules started from. It is great and one of the most
   downloaded modules on the PowerShell Gallery. It also has *a lot* of functionality that you may or may not use.
   All this functionality makes it hard to maintain. Functions in here will slowly migrate to other modules as those functions need to be updated.
-* `Carbon.Management` (this module): This module contains the core, general-purpose functions. This module will
-  frequently be a dependency of other Carbon modules.
-* `Carbon.Cryptography`: functions for encrypting and decrypting strings and creating RSA public/private keys for 
-  encrypting/decrypting.
-
+* **Carbon.Management** (this module): This module contains the core, general-purpose functions that are used by other

```

Comment:
Should there be brackets around `Carbon.Management` like the other two list items?

Reply:
That's Markdown syntax for links. When displayed in GitHub, the brackets won't show.